README.rst

```diff
@@ -232,6 +232,53 @@ would be default in next MAJOR release (unified differ, estimating, full-width p
 
     $ PHP_CS_FIXER_FUTURE_MODE=1 php php-cs-fixer.phar fix -v --diff
 
+Choose from the list of available sets:
+
+* **@PSR1**
```

Comment:
we need some descriptions for those ;)

Reply:
From where can I get the descriptions? It's a generated list.

Reply:
List of rules is also generated.
That's the point, there is no descriptions yet. I must admit that I don't see much value of list having only names.

Reply:
> I must admit that I don't see much value of list having only names.

The value is to have a quick tour of which groups we can use while configuring our `.php-cs`, as I described on the issue.

I may add descriptions of each groups if you want, just give me what descriptions you do want.

Reply:
>  just give me what descriptions you do want.

I would say that it's the biggest part to be done ;)
as I said, we don't have them ready anywhere

Reply:
I may add some description to see, but the names are clear enough IMHO.

Reply:
so, let us wait for more opinions here ;)